src/styles/TopNavStyles.ts

```diff
@@ -0,0 +1,50 @@
+import styled from 'styled-components';
+import AppBar from '@material-ui/core/AppBar';
+import IconButton from '@material-ui/core/IconButton';
+import Toolbar from '@material-ui/core/Toolbar';
+import Typography from '@material-ui/core/Typography';
+
+export const Root = styled.div`
+    flex-grow: 1;
+`;
+
+export const Bar = styled(AppBar)`
+    position: sticky;
+    background: transparent;
+    box-shadow: none;
+`;
+
+export const ToolBar = styled(Toolbar)`
```

Comment:
I actually tried just renaming it from ToolBar to smth like StyledToolbar and it worked w/o the `&&`!